vm/src/hint_processor/builtin_hint_processor/garaga.rs

```diff
@@ -14,7 +14,7 @@ use super::hint_utils::{get_integer_from_var_name, insert_value_from_var_name};
 /// x = ids.x,
 /// ids.bit_length = x.bit_length()
 /// ```
-pub fn get_felt_bitlenght(
+pub fn get_felt_bitlength(
```

Comment:
Hi @tcoratger. Because this method is public, you'll need to update the changelog, as the rename is a breaking change.